cosmic_ray/operators/remove_decorator.py

```diff
@@ -0,0 +1,27 @@
+import ast
+
+from .operator import Operator
+
+
+class RemoveDecorator(Operator):
+    """An operator that removes each of the non standard decorators."""
+    regular_decorators = ["classmethod", "staticmethod", "abstractmethod"]
+
+    def visit_FunctionDef(self, node):  # noqa
+        decorator_candidates = [x for x in node.decorator_list if \
```

Comment:
Sorry, my spacing was off in the code I posted. I've updated it. I definitely agree with PEP8 that we shouldn't extend beyond 80 chars.